pkg/cmd/auth/status/status.go

```diff
@@ -314,31 +344,36 @@ type buildEntryOptions struct {
 	username    string
 }
 
-func buildEntry(httpClient *http.Client, opts buildEntryOptions) Entry {
-	tokenIsWriteable := authTokenWriteable(opts.tokenSource)
+func buildEntry(httpClient *http.Client, opts buildEntryOptions) authEntry {
+
+	entry := authEntry{
+		Active:      opts.active,
+		Host:        opts.hostname,
+		Login:       opts.username,
+		TokenSource: opts.tokenSource,
+		Token:       opts.token,
+		GitProtocol: opts.gitProtocol,
+
+		showToken: opts.showToken,
```

Comment:
Now that we're refactoring, I think it's best to drop display-related concerns from `authEntry`.